README.md

```diff
@@ -1 +1,103 @@
-# Group-Capstone-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Group Capstone 
+
+In this project we worked collaboratively as a Group to implement a web app wich shows informations about movie shows. 
+
+The app fetches data from the Tvmaze Api,  and users can interactively add likes to their favourite shows. 
+
+The greatest features is making reservations and making comments and more so that we can read other people comments about the movie show. 
+
+If you like what you see and have a project you need coded, don't hesitate to contact us.
+
+
+# Screenshot
+![](src/groupCapstone.png) 
+
+[video presentation](https://drive.google.com/file/d/19XV32Kyiy6z-5QU6gO0r5RLOhZt1G9ZI/view?usp=sharing)
+
+
+
+# Built With
+
+- Major languages: HTML,CSS and Javascript
+- Technologies used: Web services, webpack, API
+
+
+# Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+## Prerequisites 
+- A working Laptop or Desktop
+## Setup
+- Install code editor
+- Install Git bash
+- Install webpack
+## Install
+- clone the [project](https://github.com/mutinhiri/Group-Capstone-).
+## Usage
+- Test your GitHub flow, a linter and professional documentation ability
+## Run tests
+- Run npm run build to rebuild the project
+- Run npm run start to run the project
+## Deployment
+- Push the project to your GitHub repo
+- Navigate to page in the project settings and publish.
+
+
+### Run tests
+
+Javascript linters
+
+```
+    npx eslint .
+```
+
+HTML linters
+
+```
+    npx hint .
+
+```
+
+CSS linters
+
+```
+    npx stylelint "**/*.{css,scss}"
+```
+
```

Comment:
- [ ] In this section, still we are missing the main command to test the application, please include them and make sure all tests are passing. [Hint: `npm test`]